src/lua/log.lua

```diff
@@ -511,30 +407,16 @@ local function check_cfg(cfg)
     end
 
     if cfg.format ~= nil then
-        local ok, msg = verify_option('format', cfg.format, cfg)
-        if not ok then
-            local m = "log.cfg: \'%s\' %s"
-            error(m:format('format', msg))
-        end
+        check_format(cfg.format)
     end
 
     if cfg.level ~= nil then
-        local ok, msg = verify_option('level', cfg.level)
-        if not ok then
-            local m = "log.cfg: \'%s\' %s"
-            error(m:format('level', msg))
-        end
-        -- Convert level to a numeric value since
-        -- low level api operates with numbers only.
-        if type(cfg.level) == 'string' then
-            assert(log_level_keys[cfg.level] ~= nil)
-            cfg.level = log_level_keys[cfg.level]
-        end
+        cfg.level = check_level(cfg.level)
     end
 
     if cfg.nonblock ~= nil then
         if type(cfg.nonblock) ~= 'boolean' then
-            error("log.cfg: 'nonblock' option must be 'true' or 'false'")
+            box.error(box.error.cfg, 'log nonblock', "must be 'true' or 'false'")
```

Comment:
There's a typo in this code:

```
tarantool> box.error(box.error.cfg, 'log nonblock', "must be 'true' or 'false'")
---
- error: 'box.error(): bad arguments'
...
```

Apparently, we lack tests that check errors raised on invalid log configuration :-(